pkgs/by-name/md/mdns-reflector/package.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, unstableGitUpdater, cmake, lib }: stdenv.mkDerivation rec {
+  pname = "mdns-reflector";
+  version = "unstable-2023-09-15";
```

```suggestion
  version = "0.0.1-unstable-2023-09-15";
```

[package versions must start with a digit](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)

---

This is not how unstableGitUpdater create the version numbers

---

hmm that's a good point, as far as i know that's a new policy